devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy

```diff
@@ -49,7 +49,7 @@ metadata {
 			])
 		}
 		section {
-			input "tempOffset", "number", title: "Temperature offset", description: "Select how many degrees to adjust the temperature.", range: "*..*", displayDuringSetup: false
+			input "tempOffset", "number", title: "Temperature offset", description: "Enter how many degrees to adjust the temperature.", range: "-100..100", displayDuringSetup: false
```

Comment:
Please see both my comments on https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/20469 which also apply to this PR.